entrypoint.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+cat >/etc/motd <<EOL
+  _____
+  /  _  \ __________ _________   ____
+ /  /_\  \\___   /  |  \_  __ \_/ __ \
+/    |    \/    /|  |  /|  | \/\  ___/
+\____|__  /_____ \____/ |__|    \___  >
+        \/      \/                  \/
+A P P   S E R V I C E   O N   L I N U X
+Documentation: http://aka.ms/webapp-linux
+NodeJS quickstart: https://aka.ms/golang-qs
+EOL
+cat /etc/motd
+
+service ssh start
+
+# Get environment variables to show up in SSH session
+eval $(printenv | awk -F= '{print "export " $1"="$2 }' >> /etc/profile)
+
+executable=$(find -type f -executable -exec sh -c "file -i '{}' | grep -q 'x-executable; charset=binary'" \; -print -quit)
+if [ -n "$executable" ]; then
```

Comment:
Doesn't appear that we actually run it here?

Reply:
yes

---
In reply to: [166157431](https://github.com/Azure-App-Service/go/pull/1#discussion_r166157431) [](ancestors = 166157431)